core/sequencer/server.go

```diff
@@ -403,6 +403,7 @@ func (s *Server) ApplyRevision(ctx context.Context, in *spb.ApplyRevisionRequest
 		return nil, err
 	}
 
+	reduceStart := time.Now()
```

Comment:
Probably needs a less "reducey" name, as it includes some other steps in addition to reduce. Maybe just `start` or `startedAt`?